### PR TITLE
Sambranch

### PR DESCRIPTION
**Pull request by samlsmith424**
## Features
1.  Initial tests for job service and house service with webmock
2. Controllers, facades, and services set up for jobs and houses

## Concerns
1. May need to change house api

## Pre-Submittal Checklist
- [x] All tests pass on local
- [x] Visually confirm view functionality
- [x] No prys or save_and_open_page commands remain
- [x] Unnecessary commented code removed

## As a GIF, how does this PR make you feel?
cat

### CODE REVIEW

**Review by RichardLaBrecque — CHANGES_REQUESTED** [review]
Jobs controller note:
The jobs index is generating an @houses object, suggest changing to @jobs for user empathy

**Comment by samlsmith424** [comment]
> Jobs controller note: The jobs index is generating an @HouseS object, suggest changing to @jobs for user empathy

Great looking out, it has been updated.

**Review by RichardLaBrecque — APPROVED** [review]
looks much better,  only other concern is,
Where are you assigning the city, state variables for the jobs controller? are those params

**Review by aedanjames — COMMENTED** [review]
Looks good, in the house_service_spec, are we webmock stubbed temporarily until we get a key to access the house API? Fine either way, just curious.

**Review by aedanjames — APPROVED** [review]

File: app/controllers/houses_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class HousesController < ApplicationController
+  def index
```
Comment (Eldridge-Turambi) [inline app/controllers/houses_controller.rb:2]: we might need to define what price is 🙏

File: app/services/house_service.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class HouseService
+  def self.conn
+    Faraday.new(url: "https://api.gateway.attomdata.com/")
```
Comment (Eldridge-Turambi) [inline app/services/house_service.rb:3]: is there an api key? for the self.conn method?

**Review by Eldridge-Turambi — APPROVED** [review]
💫

**Pull request merged**